.github/workflows/docs-preview.yml

```diff
@@ -16,6 +16,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-check-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
This should have a different cache key. Otherwise, they race and if one does less work that gets cached.

Reply:
Although we do make python-build and make wasm-build in both so maybe its fine?

Reply:
Why would the one that does less work get cached? Wouldn't the tortoise win the race?